network-store-server/src/main/resources/db/changelog/changesets/changelog_20240924T110000Z.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="0" author="lesoteti">
+        <sqlFile
+                dbms="postgresql"
```

Comment:
As you don't use postgres specific instructions in fix_regulationPoint_20240924T110000Z.sql, I would remove that ? This is used to specify that this changelog should only be used with postgres database.